components/dot_menu/index.ts

```diff
@@ -62,6 +64,10 @@ type Props = {
 
 const getPostMenuBindings = makeAppBindingsSelector(AppBindingLocations.POST_MENU_ITEM);
 
+const getRHSPostMenuBindings = makeRHSAppBindingSelector(AppBindingLocations.POST_MENU_ITEM);
+
+const getBindings = makeGetBindings(AppBindingLocations.POST_MENU_ITEM);
```

Comment:
`get*` is being used in two different ways here, for selectors and actions. Maybe we should change references of `get` to `fetch` when we are fetching bindings from the server?